app/controllers/mixins/ems_common_angular.rb

```diff
@@ -535,6 +543,16 @@ def set_ems_record_vars(ems, mode = nil)
                              :path     => metrics_database_name }
       end
 
+      if ems.kind_of?(ManageIQ::Providers::Kubevirt::InfraManager)
+        default_endpoint = {
+          :role                  => :default,
+          :hostname              => hostname,
+          :port                  => port,
+          :verify_ssl            => params[:default_tls_verify] == 'on' ? 1 : 0,
```

Comment:
We can use the same constants here as well: OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE